CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to the Sample Plug-in for Zowe CLI will be documented in thi
 
 ## Recent Changes
 
-- Enhancement: Added plugin lifecycle example.
+- Enhancement:
```

Comment:
I would add pr or issue links to both of these enhancement items

Reply:
good catch thank you!!